java/src/jmri/jmrit/roster/RosterEntry.java

```diff
@@ -537,7 +537,14 @@ public void setDateModified(@Nonnull String date) throws ParseException {
         try {
             // parse using ISO 8601 date format(s)
             this.setDateModified(new ISO8601DateFormat().parse(date));
-        } catch (ParseException | IllegalArgumentException ex) {
+        } catch (ParseException ex) {
+            log.debug("ParseException in setDateModified");
+            // parse using defaults since thats how it was saved if saved
+            // by earlier versions of JMRI
+            this.setDateModified(DateFormat.getDateTimeInstance().parse(date));
+        } catch (IllegalArgumentException ex2) {
+            // warn that there's perhaps something wrong with the classpath
+            log.error("IllegalArgumentException in RosterEntry.setDateModified - this may indicate a problem with the classpath, specifically multiple copies of the 'jackson` library. See release notes" );
```

Comment:
This *does* indicate a problem with the classpath; there is no ambiguity that this error indicates that the wrong jackson-databind JAR is being used.